new/components/restservice/scripts/install_plugins.py

```diff
@@ -0,0 +1,78 @@
+import tarfile
+import tempfile
+import os
+from StringIO import StringIO
+
+import fabric
+import fabric.api
+import jinja2
+
+from cloudify import ctx
+
+
+def install_plugins():
+
+    root_dir = os.path.join(ctx._endpoint.storage.resources_root,
+                            os.path.dirname(__file__))
+    install_plugins_script = 'install_plugins.sh'
+    ctx.logger.info('Installing plugins')
+
+    # Getting all the required plugins
+    plugins = ctx.node.properties.get('plugins', {})
+
+    # Shame to do all the work for nothing
+    if plugins:
+
+        # # Get components utils (for pip install)
+        with open(os.path.normpath(
```

Comment:
you can simply do `utils_content = ctx.get_resource('components/utils')` instead of all this